### DISCUSSION TITLE
Displaying of German "Umlaute" (ö,ä etc.)

### DISCUSSION THREAD

**Discussion by ghost** [chrisant996/clink#276] [Q&A] [answered]
Hello, is there an option in Clink to display the special characters like Ä, ö etc. correctly?

![2022-04-22_09h23_48](https://user-images.githubusercontent.com/98451129/164627154-52c258aa-5079-481a-b5e9-b055ef5ce113.png)

**Comment by chrisant996** [comment, marked as answer]
That is output from the `streams` program, not from Clink.  This question isn't about Clink.

**Comment by chrisant996** [comment]
Also:

- `dir /r` displays NTFS streams and displays non-ASCII characters correctly.
- Have you tried `AlternateStreamView` from NirSoft?  It is translated into many languages, so chances are good that it handles non-US characters correctly.
- Or PowerShell:
  - List streams: `Get-Item "filename" -Stream *`
  - Delete streams: `Remote-Item -Path "filename" -Stream "stream_name"` (I don't know if it supports wildcards; you could try and find out).
